frontend/dockerfile/linter/ruleset.go

```diff
@@ -0,0 +1,110 @@
+package linter
+
+import (
+	"strings"
+
+	"github.com/moby/buildkit/frontend/dockerfile/parser"
+)
+
+const (
+	RuleStageNameCasing = "StageNameCasing"
+	RuleFlagCasing      = "FlagCasing"
```

Comment:
`RuleFlagCasing` was a mistake I think. Looks like the flags are already case sensitive.